darts/tests/models/forecasting/test_regression_models.py

```diff
@@ -663,6 +663,14 @@ def test_models_accuracy_multiseries_multivariate(self):
             self.multivariate_multiseries_accuracies,
         )
 
+    def test_min_train_series_length(self):
+        model = self.models[3](lags=2)
```

Comment:
Thanks!
Could you maybe now just
* Invoke the real constructors (e.g., `CatBoostModel(lags=2)`), so we avoid relying on the order in the list (which could change in the future), and
* Also test the case when the `lags` is specified as a list of integers?

Thanks!

Reply:
done